src/components/POLine/FundDistribution/FundDistributionForm.js

```diff
@@ -35,8 +37,9 @@ class FundDistributionForm extends Component {
   };
 
   renderSubForm = (elem, index, fields) => {
-    const { funds } = this.props;
+    const { funds = [] } = this.props;
     const fund = fields.get(index);
+    const fundCode = get(funds.find(({ value }) => value === fund.fundId), 'code');
```

Comment:
desn't `fund` above contain all needed information?

Reply:
currently fund contains only ```fundId``` and ```percentage``` as formValues part